File: python_leaoai/python_leaoai/services/ApplicationServices/Configs/ConfigsService.py
# framework/services/configs_service.py

from typing import Any
from framework.repositories.repository import Repository
from framework.services.generic_service import GenericService
from data.models.core import HeadConfig
from framework.services.item_service import ItemService
from sqlalchemy.ext.asyncio import AsyncSession

class ConfigsService(GenericService[HeadConfig]):
    def __init__(
        self,
        session: AsyncSession,
        item_service: ItemService,
        repository: Repository[HeadConfig]
    ):
        super().__init__(session, item_service, repository)
        self._session = session
        self._srv_item = item_service
        self._repository = repository

File: Framework.Services/ApplicationServices/Configs/ConfigsService.cs
using Data;
using Data.Models.Core;
using Framework.Repositories;
using Framework.Services.ApplicationServices.Core;

namespace Framework.Services.ApplicationServices.Configs
{
    public class ConfigsService : GenericService<HeadConfig, IRepository<HeadConfig>>
    {
        private readonly ApplicationDbContext _context;
        private readonly ItemService _srvItem;
        private readonly IRepository<HeadConfig> _repository;

        public ConfigsService(ApplicationDbContext context, ItemService itemService, IRepository<HeadConfig> repository) : base(context, itemService, repository)
        {
            _context = context;
            _srvItem = itemService;
            _repository = repository;
        }
    }
}
